chia/types/spend_bundle_conditions.py

```diff
@@ -1,30 +1,3 @@
-from dataclasses import dataclass
-from typing import List, Optional, Tuple
+from chia_rs import Spend, SpendBundleConditions
```

Comment:
the type stubs for these classes are here: https://github.com/Chia-Network/chia_rs/blob/main/wheel/chia_rs.pyi